src/content/en/2022/performance.md

```diff
@@ -1,22 +1,989 @@
+
+The second most popular LCP element type is `div`. This is a generic HTML container that could be used for text or styling background images. To help disambiguate how often these elements contain images or text, we can evaluate the `url` property of the [LCP API](https://developer.mozilla.org/en-US/docs/Web/API/LargestContentfulPaint).  According to the <a hreflang="en" href="https://www.w3.org/TR/largest-contentful-paint/#dom-largestcontentfulpaint-url">specification</a>, when this property is set, the LCP content must be an image.
+
+{{ figure_markup(
+  image="top-lcp-content-types.png",
+  caption="The percent of pages that use each type of LCP content.",
+  description="Bar chart showing an image is the LCP content type on 82% of desktop pages and 72% of mobile pages, it is text on 17% and 26% respectively, and an inline image on 2% of desktop and 1% of mobile pages.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vR-dJP3uphZoGE5A_luniNBFm5V2ww6irfOxANg0hrMid7gjgrtchsN_utOIDOvVZUjIwpmUBb27nHF/pubchart?oid=839846485&format=interactive",
+  sheets_gid="872701281",
+  sql_file="lcp_resource_type.sql"
+  )
+}}
+
+We see that 72% of mobile pages and 82% of desktop pages have images as their LCP. For example, these images may be traditional `img` elements or CSS background images. 26% of mobile pages and 17% of desktop pages have text-based content as their LCP.
+
+1% of pages actually use inline images as their LCP content. This is almost always a bad idea for a number of reasons, mostly around caching and complexity.
+
+#### LCP prioritization
+
+After the HTML document is loaded, there are two major factors that affect how quickly the LCP resource itself can be loaded: discoverability and prioritization. We'll explore <a hreflang="en" href="#lcp-static-discoverability">LCP discoverability</a> later, but first let's look at how LCP images are prioritized.
+
+Images are not loaded at high priority by default, but thanks to the new <a hreflang="en" href="https://web.dev/priority-hints/">Priority Hints</a> API, developers can explicitly set their LCP images to load at high priority to take precedence over non-essential resources.
+
+{{ figure_markup(
+  content="0.03%",
+  caption="The percent of pages that use `fetchpriority=high` on their LCP element.",
+  classes="big-number",
+  sheets_gid="600760184",
+  sql_file="lcp_element_data_2.sql"
+  )
+}}
+
+0.03% of pages use `fetchpriority=high` on their LCP elements. Counterproductively, a handful of pages actually _lower_ the priority over their LCP images: 77 pages on mobile and 104 on desktop.
+
+`fetchpriority` is still very new and not supported everywhere, but there's little to no reason why every developer shouldn't be using it. [Patrick Meenan](https://twitter.com/patmeenan), who helped develop the API, [describes it](https://twitter.com/patmeenan/status/1460276602479251457) as a "cheat code" given how easy it is to implement relative to the potential improvements.
```

Comment:
I disagree. See [this thread](https://webperformance.slack.com/archives/C04BK7K1X/p1666389089163239) where @philipwalton and I explain when it might not be a good idea to do it.

I think we should language like "there's little to no reason why every developer shouldn't be using it" when there clearer are cases of this.

Reply:
WDYT about rephrasing to "there's little to no reason why it shouldn't be in every developer's toolbox"? The point I'm making is that everyone should use it responsibly, not necessarily that it should be blindly used on the same image, every time.

Reply:
That works for me!